website/templates/index/quickstart.html

```diff
@@ -20,10 +20,10 @@ <h2 id="quick-start" class="display-2 mb-5 mb-md-8 mb-lg-10 text-center">Quick S
                     <a class="nav-link" id="arm-tab" data-toggle="tab" href="#arm" role="tab" aria-controls="arm" aria-selected="false" title="ARM packages">Linux (ARM)</a>
                 </li>
                 <li class="nav-item">
-                    <a class="nav-link" id="mac-x86-tab" data-toggle="tab" href="#mac-x86" role="tab" aria-controls="mac-x86" aria-selected="false" title="Mac x86 packages">Mac (x86)</a>
+                    <a class="nav-link" id="mac-x86-tab" data-toggle="tab" href="#mac-x86" role="tab" aria-controls="mac-x86" aria-selected="false" title="Mac x86 packages">Mac (Intel)</a>
```

Comment:
```suggestion
                    <a class="nav-link" id="mac-x86-tab" data-toggle="tab" href="#mac-x86" role="tab" aria-controls="mac-x86" aria-selected="false" title="Mac x86 packages">macOS (Intel)</a>
```

Reply:
Why `macOS` instead of `MacOS`?

Reply:
https://fortune.com/2016/06/13/apple-mac-os-sierra/

Reply:
https://www.apple.com/macos/monterey/